### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Binomial.lean [hunk 1]
```diff
@@ -406,15 +406,9 @@ theorem choose_zero_pos (R) [NonAssocRing R] [Pow R ℕ] [NatPowAssoc R] [Binomi
...
-  rw [eq_ite_iff]
-  by_cases hk: k = 0
-  constructor
-  · rw [hk, choose_zero_right, ← Prod.mk.inj_iff]
-  · right
-    constructor
-    · exact hk
-    · rw [← @Nat.le_zero, Nat.not_le] at hk
-      rw [choose_zero_pos R hk]
+  split_ifs with hk
```
Comment (grunweg) [inline Mathlib/RingTheory/Binomial.lean:409]: Nice golf!

File: Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean [hunk 2]
```diff
@@ -56,8 +56,10 @@ theorem mem_nonZeroDivisors_of_constantCoeff {φ : MvPowerSeries σ R}
   rw [Finset.sum_eq_single (e,0), coeff_zero_eq_constantCoeff]
   · rintro ⟨u, _⟩ huv _
     suffices u < e by simp only [he u this, zero_mul, map_zero]
-    have hue : u ≤ e := by simp only [← mem_antidiagonal.mp huv, le_add_iff_nonneg_right, zero_le]
-    exact ⟨hue, fun h ↦ by simp_all [mem_antidiagonal.mp huv, le_antisymm hue h]⟩
+    apply lt_of_le_of_ne
```
Comment (grunweg) [inline Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean:59]: I'm not sure if I understand this change: let's have another pair of eyes on it.

**Review by grunweg — COMMENTED** [review]
Looks very nice, as usual - thanks!.
One change I couldn't quite understand, perhaps another reviewer or maintainer does:
maintainer merge

File: Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean (hunk 2 above)
Comment (riccardobrasca) [reply Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean:59]: It doesn't introduce a new hypothesis, why not.

**Review by riccardobrasca — APPROVED** [review]
Thanks!

bors merge

**Pull request closed without merging**